application-workloads/visualstudio/az-400-dev-env/azuredeploy.json

```diff
@@ -0,0 +1,263 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "storagename": {
```

Comment:
since this is a lab scenario just make this a variable

Reply:
Yes, thanks for suggestion.